src/transformers/modeling_utils.py

```diff
@@ -2013,18 +2124,22 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         config.name_or_path = pretrained_model_name_or_path
 
         # Instantiate model.
+        init_contexts = [no_init_weights(_enable=_fast_init)]
```

Comment:
if `low_cpu_mem_usage = True` then `no_init_weights` is not needed no? As far as I understand when `low_cpu_mem_usage=True`, then all weights will be either `meta` or pretrained weights and no init will happen anyways no? But guess it also doesn't hurt

Reply:
It doesn't really hurt, but it shouldn't be needed, yes.